### ISSUE TITLE
ag-grid drag n drop ghost element

### ISSUE THREAD

**Issue by bwinchester** [ceolter/ag-grid-enterprise#56]
I have an enterprise license and our app has a BODY height of 100%, and an element that takes up most of the body has thousands of pixels of height. Your logic to position the draggable columns into the Group Tray started failing after moving from 4.X to 5.1.2 of AG-GRID. Specifically, the dragged header column item top position (if the page is scrolled lower/more than the window height (what you are returning for getBodyHeight in chrome)). The Body height calculation is wrong in Chrome, what is being returned is the window viewable height, if you indeed want the 'full' body height. I can reproduce this in Chrome latest on Windows and OSX.

Then again, why worry about that at all? Can you assume if you have a drag event, then your browser has height. And top should just equal the mouseEvent.pageY  since the dnd-ghost element is absolutely positioned.

```
if (browserHeight > 0) {
    top = event.pageY;     
}
```

REF CODE:
[positionGhost method](https://github.com/ceolter/ag-grid/blob/c3dcfb7bf6cd6fe8629aa7ddc25de202efd2ef08/src/ts/dragAndDrop/dragAndDropService.ts#L302)

jQuery Document Height code if you are looking for something to return the full body height:
```
    return Math.max(
      document.body[ "scrollHeight"], document.documentElement[ "scrollHeight"]],
      document.body[ "offsetHeight"], document.documentElement[ "offsetHeight"],
      document.documentElement[ "clientHeight"]);
```

**Comment by ceolter** [comment]
i am aware of a bug that was not taking into consideration the browser vertical scroll position - it's fixed in 5.2.x - is that the cause of this problem? i'm about to do a release 5.2.x, can you verify if the problem persists when moving to 5.2.x?

**Comment by bwinchester** [comment]
Indeed, checked out the code update to drag n drop and can verify it fixes this issue. Glad you were on it and its in the change log already. BZ.
